rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -929,10 +929,20 @@ private void requestRelease(Address destinationAddress, co.rsk.core.Coin release
                 provider.getReleaseRequestQueue().add(destinationAddress, valueToRelease);
             }
 
+            RskAddress sender = rskTx.getSender(signatureCache);
             if (activations.isActive(ConsensusRule.RSKIP185)) {
-                eventLogger.logReleaseBtcRequestReceived(rskTx.getSender(signatureCache).toHexString(), destinationAddress, valueToRelease);
+                eventLogger.logReleaseBtcRequestReceived(
+                    sender,
+                    destinationAddress,
+                    releaseRequestedValue
+                );
             }
-            logger.info("[requestRelease] releaseBtc successful to {}. Tx {}. Value {}.", destinationAddress, rskTx, releaseRequestedValue);
+            logger.info(
+                "[requestRelease] releaseBtc successful to {}. Tx {}. Value {}.",
+                destinationAddress,
+                rskTx,
```

Comment:
```suggestion
                rskTx.getHash(),
```

Should it be enough by only printing the tx hash here? At this moment it will print something like:

```
[hash=97c7e1a4bdabb444c883d1eeac4eb29cc98a91d75ab386b9229cb34743b8a315  nonce=, gasPrice=100, gas=03e8, receiveAddress=0000000000000000000000000000000001000006, value=50000000000000000000, data=80af2871, signatureV=27, signatureR=01e1d5704efc6c67f6e797ca104d46b7a9711bfdd4ceb2660af4d2cf29ec145f, signatureS=34d003d60ccb9bc20fead0d3d57a05e15d5ac748d6e0377047b8a9008910dec9]
```
Is that what we want? What do you think?

Reply:
Working on changing this in another refactor